extensions/github1s/src/interfaces/github-api-rest.ts

```diff
@@ -89,7 +89,7 @@ export const validateToken = (token: string) => {
 // [List matching references](https://docs.github.com/en/rest/reference/git#list-matching-references)
 // can returned all branches for a request, and there is an issue for this API
 // https://github.com/github/docs/issues/3863
-export const getGithubBranchRefs = (owner: string, repo: string) => {
+export const getGitHubBranchRefs = (owner: string, repo: string) => {
```

Comment:
It isn't a big deal for me to drop changes to things (usually by commit = word, but also by file path).